src/components/coins-data-display/CryptoDisplay.component.tsx

```diff
@@ -2,10 +2,23 @@ import React, { useEffect, useState } from 'react';
 import Coinlist from './coinlist/Coinlist.component';
 import { CoinAPI } from '../../services/coingecko/coingeckoApi.service';
 import { ICoin } from '../../models/coin.interface';
+import { TiSortAlphabetically } from 'react-icons/ti'
+import { FaMoneyBillAlt } from 'react-icons/fa';
+import { BsGraphUp } from 'react-icons/bs';
+import { GrMoney } from 'react-icons/gr';
+import { IconContext } from "react-icons";
+
+const enum Column {
```

Comment:
A file for enums could be created where all enums in future are placed

Reply:
import React, { useEffect, useState } from 'react';
=> import * as React from 'react';
React.useState()

Prefer this but sometimes i do like this also